### ISSUE TITLE
NotepadStarter.exe places a lock on files previously open by notepad++

### ISSUE THREAD

**Issue by garyturnas** [lygstate/NotepadStarter#2]
NotepadStarter.exe places a lock on files previously open by notepad++.  As a result removing or changing the name of the folder that contained the file(s) view was not possible. I had to use process explorer to determine the app that was causing the lock.
I notice several instances of NotepadStarter.exe running in task manager when the notepad++ app was not open.
I believe that  NotepadStarter.exe is not closing once notepad++ is exit/close, even if the file view was deleted or no longer exist.

**Comment by lygstate** [comment]
Yes, I know what's happening, when the file closed, the tooltip doesn't closed.
I'll investigate that.
